### PR TITLE
wrapper.js

### PR DESCRIPTION
**Pull request by ChiefChippy2**
new cmd where !wrapper player ChiefChippy2 would imitate a call to the api, like getPlayer("ChiefChippy2")
inb4 very unstable.

### CODE REVIEW

**Comment by StavZ** [comment]
`TypeError: Found non-callable @@iterator` at line 41:46

**Comment by ChiefChippy2** [comment]
args.args

**Comment by StavZ** [comment]
image
If -raw flag

File: src/commands/Hypixel/wrapper.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  async exec (message, args) {
+    if (!args.method) return message.channel.send('I need to know which method to call.');
+    args.args = args.args.split(/ +/);
+    if (args.args.length > 2 && args.method !== 'getGuild') return message.channel.send('Too many arguments. Only 1 is allowed, except for getGuild');
+    args.args.shift();
+    const result = await hypixel[args.method](...args.args).catch(e => {
+      return message.channel.send(`Error occurred: \`${e}\``);
+    });
+    if (args.noParse) {
+      return message.channel.send('API Result : ', { files: [JSON.stringify(result)] });
```
Comment (ChiefChippy2) [inline src/commands/Hypixel/wrapper.js]:
```suggestion
      return message.channel.send('API Result : ', { files: [Buffer.from(JSON.stringify(result))] });
```

Comment (ChiefChippy2) [reply src/commands/Hypixel/wrapper.js]: bruh gh mobile sucks cant even submit a review properly

**Pull request merged**